wpiutil/build.gradle

```diff
@@ -219,7 +219,8 @@ model {
 }
 
 dependencies {
-    compile fileTree(dir: "thirdparty", includes: ["*.jar"])
+    compile "org.ejml:ejml-simple:0.38"
```

Comment:
Is this a place where we should use `api` instead of `compile`?

Reply:
Replacing `compile` with `api` makes configuring fail on my machine. 
```
> Could not find method api() for arguments [org.ejml:ejml-simple:0.38] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

Reply:
API is for if using the java-library plugin which we don't. And this wouldn't be the pr to change that.

Reply:
ah. ok